src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin square ($size) {
+  width: $cell-size;
```

Comment:
The square mixin should use the passed $size parameter instead of the $cell-size variable.